invenio.cfg

```diff
@@ -414,51 +413,63 @@ RDM_FILES_DEFAULT_MAX_FILE_SIZE = 50 * 10 ** 9  # 50GB
-                                      "inspire": {"label": _("Inspire"),
-                                                  "validator": schemes.is_inspire,
-                                                  "datacite": "INSPIRE"},
-                                      "inis": {"label": _("INIS"),
-                                               "validator": schemes.is_inspire,
-                                               "datacite": "INIS"
-                                               },
-
-                                      "indico": {
-                                            "label": _("Indico"),
-                                            "validator": schemes.is_indico,
-                                            "datacite": "INDICO"
-                                        },
-                                               },
-                                      "hdl": {
-                                          "label": _("Handle"),
-                                          "validator": schemes.is_handle,
-                                          "datacite": "HANDLE",
-                                      },
-                                   }
-
-RDM_RECORDS_PERSONORG_SCHEMES = {**RDM_RECORDS_PERSONORG_SCHEMES,
-                                 **{"inspire_author": {"label": _("Inspire"),
-                                                       "validator": schemes.is_inspire_author,
-                                                       "datacite": "INSPIRE"},
-                                    "lcds": {"label": _("CDS"),
-                                             "validator": schemes.is_legacy_cds,
-                                             "datacite": "CDS"}
-                                    }
-                                 }
+RDM_RECORDS_IDENTIFIERS_SCHEMES = {
+    "cdsrn": {"label": _("CDS Reference"),
+              "validator": always_valid,
+              "datacite": "CDS"},
+    "aleph": {"label": _("Aleph number"),
+              "validator": schemes.is_aleph,
+              "datacite": "ALEPH"},
+    "lcds": {"label": _("CDS"),
+             "validator": schemes.is_legacy_cds,
+             "datacite": "CDS"
+             }
+}
+
+RDM_RECORDS_RELATED_IDENTIFIERS_SCHEMES = {
+    **RDM_RECORDS_RELATED_IDENTIFIERS_SCHEMES,
+    **{
+        "inspire": {"label": _("Inspire"),
+                    "validator": schemes.is_inspire,
+                    "datacite": "INSPIRE"},
+        "inis": {"label": _("INIS"),
+                 "validator": schemes.is_inspire,
+                 "datacite": "INIS"
+                 },
+
+        "indico": {
+            "label": _("Indico"),
+            "validator": schemes.is_indico,
+            "datacite": "INDICO"
+        },
+        "hdl": {
+            "label": _("Handle"),
+            "validator": schemes.is_handle,
+            "datacite": "HANDLE",
+        },
+        "lcds": {"label": _("CDS"),
+                 "validator": schemes.is_legacy_cds,
```

Comment:
On a side note, could we simplify this function a bit?

https://github.com/CERNDocumentServer/cds-rdm/blob/13ddbd428f01495c950780d7ab16c9bb1a3921b0/site/cds_rdm/schemes.py#L91-L94

This is just equivalent to `val.isdigit()` right? Not sure if there is a performance difference but I think the regex is a bit less readable. It's not super important right now though